### PR TITLE
14996 dev info in head

### PR DESCRIPTION
**Pull request by jtmst**
## Description
Relates to #[14996](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14996). (or closes?)

## Testing done
Local vision testing

## Screenshots
image

## QA steps
- Navigate to a story listing, story, QA or personal profile page
- Verify that comment appears with relevant info

## Is this PR blocked by another PR?
- Add the `DO NOT MERGE` label
- Add links to additional PRs here:

### CODE REVIEW

File: src/templates/globals/util/HTMLComment.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { useEffect } from 'react'
+
+interface HTMLCommentProps {
+  position: 'head' | 'footer'
```
Comment (tjheffner) [inline src/templates/globals/util/HTMLComment.tsx:4]: i know these are the only two places we've had to use comments, but it feels kind of limiting. i'd really prefer something more generic... but we can cross that bridge when we come to it, this should be fine for our purposes for now.

**Review by tjheffner — CHANGES_REQUESTED** [review]

File: src/pages/[[...slug]].tsx [hunk 2]
```diff
@@ -15,6 +15,7 @@ import { StoryListing } from '@/templates/layouts/storyListing'
 import { QuestionAnswer } from '@/templates/layouts/questionAnswer'
 import RESOURCE_TYPES from '@/lib/constants/resourceTypes'
 import { isListingPageSlug } from '@/lib/utils/listingPages'
+import HTMLComment from '../templates/globals/util/HTMLComment'
```
[the remark this answers is not in the document]
Comment (jtmst) [reply src/pages/[[...slug]].tsx:18]: good call, fixed

File: src/data/queries/newsStory.ts [hunk 3]
```diff
@@ -46,6 +46,8 @@ export const formatter: QueryFormatter<NodeNewsStory, NewsStoryType> = (
 ) => {
   return {
     id: entity.id,
+    entityId: entity.drupal_internal__nid,
+    path: entity.path?.alias,
```
[the remark this answers is not in the document]
Comment (jtmst) [reply src/data/queries/newsStory.ts]: fixed, leftover from initial testing i think

File: src/templates/globals/util/HTMLComment.tsx (hunk 1 above)
Comment (jtmst) [reply src/templates/globals/util/HTMLComment.tsx:4]: I agree that its limiting but I believe there will be some extra consideration around cleanup for adding many more comments, and to the body.  I don't think html comments are something we will need to reach for very often though.

**Pull request merged**